lib/commands/general.js

```diff
@@ -87,6 +88,18 @@ commands.setUrl = async function (url) {
   await this.adb.startUri(url, this.opts.appPackage);
 };
 
+commands.mobileDeepLink = async function (opts = {}) {
+  const {url, package:pkg} = opts;
+
+  const res = await this.adb.startUri(url, pkg);
+
+  // This command never seems to return a non-zero status code, so check for this phrase
+  // to see if it didn't work
+  if (res.toLowerCase().includes('unable to resolve intent')) {
```

Comment:
should this verification be inside `statrtUri` ?

Reply:
The `unable to resolve intent` thing only seems to happen in the case where a deep link is invoked and the package isn't installed and there's no instant app to open.... but I suppose it wouldn't hurt to just include this in the `startUri` method.